src/Operation/ListIndexes.php

```diff
@@ -19,7 +19,7 @@
 
 use EmptyIterator;
 use MongoDB\Driver\Command;
-use MongoDB\Driver\Exception\RuntimeException as DriverRuntimeException;
```

Comment:
Related to my comment below, I think we may actually need to leave this line in place to satisfy the reference in `@throws` (phpcs might complain otherwise, even though it's just in a comment). So CommandException should probably be added as a new line.